R/nitrogen_efficiency.R

```diff
@@ -24,13 +24,13 @@
 #'          
 #' @export
 calc_n_efficiency <- function(B_LU_BRP, B_SOILTYPE_AGR, B_GWL_CLASS, B_AER_CBS, A_SOM_LOI, A_CLAY_MI,
-                              D_PBI, D_K, D_PH_DELTA, D_NLV, M_GREEN = FALSE, B_FERT_NORM_FR = 1){
+                              D_PBI, D_K, D_PH_DELTA, leaching_to, M_GREEN = FALSE, B_FERT_NORM_FR = 1){
```

Comment:
Replacing `D_NLV` by `leaching_to` is a breaking change. Please inform users in the changelog